test/e2e/k8s_provisioner_test.go

```diff
@@ -70,5 +79,47 @@ var _ = Describe("k8s provisioner", func() {
 				}).Should(BeTrue())
 			})
 		})
+
+		It("should re-create underlying system resources", func() {
```

Comment:
Note: we can extend this logic to also check that the object and metadata ConfigMaps for this Bundle object get re-created. This can be done as a follow-up?

Reply:
Added a TODO comment so we don't lose track. It would be nice to differentiate between FIXME's and TODO's, where the latter has a github action that automatically creates follow-up tickets, and the former is a considered a blocker for merging the PR and mainly used in the WIP/draft PR contest. cc @joelanford in case this logic makes sense to you?